bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/GroupItem.java

```diff
@@ -251,7 +255,20 @@ public void stateChanged(Item item, State oldState, State newState) {
      */
     @Override
     public void stateUpdated(Item item, State state) {
+        oldGroupState = this.state; 
         setState(function.calculate(members));
+        sendGroupStateUpdatedEvent(item.getName(), this.state);

```

Comment:
* There is an update event that indicates that the state was updated (regardless if the state changed or not).
* There is an changed event that indicates that the state has been changed.

If I want to build a life chart, I want perhaps to see a dot for every state. If the state is not changed, I would like to see a dot because it signals that the end device is still alive and say "yes, it is still 19° C".

If I would react on state changes, I do not want to handle all "updates", I am interested in changes only. I register for changed event and not for update events. If the state changes and there is an update event posted only, but no changed event, there is something broken (to me).